### PR TITLE
Sophia's branch change

### PR DESCRIPTION
**Pull request by SoCodeDo**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```
Comment (ringolingo) [inline .idea/.gitignore:8]: THIS FILE IS FULL OF SECRETS

File: favorite-color.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+puts "teal is one of my favorite colors!"
```
Comment (ringolingo) [inline favorite-color.txt]:
```suggestion
puts "mint is one of my favorite colors!"
```

**Review by ringolingo — COMMENTED** [review]
so many files....

File: .idea/.gitignore (hunk 1 above)
Comment (kashenafi) [reply .idea/.gitignore:8]: THIS FILE MADE. MY. DAY.

File: favorite-color.txt (hunk 2 above)
Comment (kashenafi) [inline favorite-color.txt]:
```suggestion
puts "BURNT ORANGE is one of my favorite colors!"
```

**Review by kashenafi — APPROVED** [review]
SOPHIA, YOU'RE AWESOME.

**Pull request merged**